workers/unity/Packages/com.improbable.gdk.transformsynchronization/Systems/SetNewlyCreatedGameObjectTransformSystem.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Generated.Improbable;
+using Generated.Improbable.Transform;
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Core.GameObjectRepresentation;
+using Unity.Collections;
+using Unity.Entities;
+using UnityEngine;
+using Quaternion = UnityEngine.Quaternion;
+
+namespace Improbable.Gdk.TransformSynchronization
+{
+    public class SetNewlyCreatedGameObjectTransformSystem : ComponentSystem
```

Comment:
When does this system run?

Reply:
Ohh I left that in. Didn't mean to. Since I pulled the GameObjectInitSystem into a separate assembly and it has access to the SpatialOSTransform, it's still able to do what this was meant to do